cmd/check.go

```diff
@@ -40,6 +41,13 @@ var checkCmd = &cobra.Command{
 			return types.Error{Msg: "No PackageManifests returned from PackageServer."}
 		}
 
+		if cmd.Flag("list-packages").Value.String() == "true" {
```

Comment:
Just a small nit, Instead of checking the string value, we can directly use `if checkflags.ListPackages` since it is a boolean flag.